docs/documentation/stories/moving-into-the-cli.md

```diff
@@ -0,0 +1,63 @@
+# Moving your project to Angular-CLI
+
+It's easy to move your existing project to use Angular-CLI.
+The most straightforward way is to to make a new CLI project and copy over your app files.
+
+Let's see how to do it step by step.
+Here we use a project made with the [official QuickStart](https://github.com/angular/quickstart)
+as an example, but you should be able to adjust these instructions to other setups.
+
+Note for Windows users: we show unix commands here like `cp` to copy and `rm -rf` to delete files.
+Windows does not have these commands so use Explorer instead.
+
+Start with preparing your existing project folder, let's call it `awesome-app`.
+- commit and push your existing changes.
+- clean your folder from temporary files and ignored files using `git clean -fdx`.
+- rename your project folder to `old-awesome-app`.
+
+Now make a new project on the same folder as `old-awesome-app` using Angular-CLI.
+- Read up on [Angular-CLI prerequisites](https://github.com/angular/angular-cli#prerequisites).
+- Install the CLI globally: `npm install -g @angular/cli`.
+- Make a new app: `ng new awesome-app`.
```

Comment:
It is not clear that they have to go up a level before doing this. Someone might confuse it and think they are doing this inside their old app. We should make it clear they are doing this at same level as the old app.